packages/client/src/utils/locationUtils.ts

```diff
@@ -129,7 +130,7 @@ export function generateSearchableLocations(
 
     return {
       id: location.id,
-      searchableText: location.name,
+      searchableText: getLocalizedLocationName(intl, location),
       displayLabel: locationName
```

Comment:
I wonder which one of these 2 should be localized or both?

Reply:
well, when the labels are localized, wouldn't the user be searching by the localized names? 🤔

Reply:
makes sense. then what about the displayLabel? Isn't that what's actually visible to the user?

Reply:
ah okay it's already localized by the `generateLocationName` function right?

Reply:
yes, already localized